src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -651,6 +651,27 @@ Once the event template conforms to a valid JSON, you can send a test notificati
 
     Create or edit a workflow, click on the mobile notifier, select the destination, and save the workflow.
 
+  <CollapserGroup>
+    <Collapser
+    className="freq-link"
+    id="opsgeniewebhook_templates"
+    title="Opsgenie"
+  >
+    **Webhook template for Atlassian Opsgenie**
+      Send notifications from workflows to Opsgenie using using a webhook template
+     [Opsgenie webhook template for workflows](https://support.atlassian.com/opsgenie/docs/integrate-opsgenie-with-new-relic-alerts-new/)
+  </Collapser>
+  
+  
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="Splunk_on_call_templates"
+    title="Splunk On-call(VictorOps)"
+  >
+    **Webhook template for Splunk On-call**
+     Send notifications from workflows to Splunk On-call using a webhook template
+     [Splunk On-Call webhook template for workflows](https://help.victorops.com/knowledge-base/new-relic-integration-guide/#:~:text=About%20Splunk%20On-Call%20and%20New%20Relic,-Operations%20and%20development&text=With%20the%20Splunk%20On-Call,solve%20problems%20in%20real-time)
```

Comment:
the `~` in this link is throwing an error when being parsed into MDX, is there a different link we can use?